### PR TITLE
BUGFIX: SD-294: Export previous result after filter out no data

### PR DESCRIPTION
**Pull request by ptquang86**
PR for [SD-294](https://jira.intgdc.com/browse/SD-294)

Demos:
- AD: https://github.com/gooddata/gdc-analytical-designer/pull/2048
- KD: https://github.com/gooddata/gdc-dashboards/pull/1654

Check list
- [x] Change was tested using dev-release in Analytical Designer and Dashboards 
- [ ] Update CHANGELOG

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3150/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2498/ : SUCCESS in 2m 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3152/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2500/ : SUCCESS in 1m 59s

**Comment by ptquang86** [comment]
extended test - storybook

File: src/components/core/base/test/VisualizationLoadingHOC.spec.tsx [hunk 1]
```diff
@@ -178,6 +179,24 @@ describe('VisualizationLoadingHOC', () => {
         });
     });
 
+    it('should show EmptyResultError when exporting NO DATA result', async (done) => {
+        const onError = jest.fn();
```
Comment (ivan-nejezchleb) [inline src/components/core/base/test/VisualizationLoadingHOC.spec.tsx]: why it is here, when it is not tested?

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1313/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3153/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/781/ : SUCCESS in 40m 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3154/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2501/ : SUCCESS in 11m 15s

File: src/components/core/base/test/VisualizationLoadingHOC.spec.tsx (hunk 1 above)
Comment (ptquang86) [reply src/components/core/base/test/VisualizationLoadingHOC.spec.tsx]: replaced jest.mock with noop

**Comment by ivan-nejezchleb** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3163/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2510/ : SUCCESS in 2m 25s

**Comment by ptquang86** [comment]
- [ ] Change was tested using dev-release in Analytical Designer and Dashboards

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/678/ : SUCCESS in 37s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2511/ : SUCCESS in 4m 47s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1314/ : ABORTED in 1h 00m 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/659/ : SUCCESS in 33m 33s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/679/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2512/ : SUCCESS in 8m 35s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1315/ : SUCCESS in 41m 31s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/660/ : SUCCESS in 12m 31s

**Pull request merged**